Ermfish Defense Force

Govorunb opened the pull request:
Because ermfish are an ~~invasive~~ endangered species, they must be protected from the vile clutches of the players. All who do unspeakable things upon them like consuming them for their nutrition content - or, worse yet, merely because they enjoy hearing the agonized screams of the poor fish - must be punished.

Now, the ermfish are no longer helpless.

With the newly developed EDF (Ermfish Defense Force) initiative, the Erm Moon will dispatch squads of "specially trained" assassins to chase down unruly players. No tutel is safe. Retribution will be dealt swiftly and mercilessly.

Govorunb committed:
add EDF (ermfish defense force)

Govorunb committed:
add "warning"

Govorunb committed:
fix sn

Govorunb committed:
polish

save aggro value
raise the aggro gains a little bit
increase cooldown (gives more time to accumulate aggro between spawns)
stop counting defenders as active when they die

Govorunb committed:
fix savedata oopsie

Alexejhero committed:
Merge branch 'main' into edf

Alexejhero committed:
build

Alexejhero reviewed (changes requested):
![](https://cdn.7tv.app/emote/6235309d961be6c9af3d2d1f/4x.webp)

Govorunb committed:
rework

short-term karma gain/decay instead of long-term (gives immediate feedback and flows better mechanically)
spawn only ermsharks (yes chef...)
don't save aggro (since it's immediate anyway)

Govorunb committed:
fix defenders leaving when you're almost dead

(instead of fully dead)

Govorunb committed:
also remove cooldown i guess :(

Govorunb committed:
Merge branch 'main' into edf

Govorunb committed:
assets

Alexejhero committed:
Merge branch 'main' into edf

Govorunb committed:
fix decoy ermsharks eating ermfish from your hands

move kill squad spawn point a tiny bit forward if raycast blocked
